api/events/schema.py

```diff
@@ -228,34 +231,64 @@ class Mutation(graphene.ObjectType):
              event[required]")
 
 
-class PaginatedDailyRoomEvents(graphene.ObjectType):
+class PaginateEvents(graphene.ObjectType):
     """
-    Paginated result for daily room events
+        Paginate the number of events returned
     """
-    DailyRoomEvents = graphene.List(DailyRoomEvents)
-    has_previous = graphene.Boolean()
-    has_next = graphene.Boolean()
+    events = graphene.List(Events)
     pages = graphene.Int()
     query_total = graphene.Int()
+    has_next = graphene.Boolean()
+    has_previous = graphene.Boolean()
+    events = graphene.List(Events)
+    page = graphene.Int(),
+    per_page = graphene.Int(),
+
+
+# class PaginatedDailyRoomEvents(graphene.ObjectType):
+#     """
+#     Paginated result for daily room events
+#     """
+#     DailyRoomEvents = graphene.List(DailyRoomEvents)
+#     has_previous = graphene.Boolean()
+#     has_next = graphene.Boolean()
+#     pages = graphene.Int()
+#     query_total = graphene.Int()
 
 
 class Query(graphene.ObjectType):
     all_events = graphene.Field(
-        PaginatedDailyRoomEvents,
-        start_date=graphene.String(),
-        end_date=graphene.String(),
+        PaginateEvents,
         page=graphene.Int(),
         per_page=graphene.Int(),
+        start_date=graphene.String(),
+        end_date=graphene.String(),
         description="Query that returns a list of events given the arguments\
-            \n- start_date: The date and time to start selection \
-                            when filtering by time period\
-            \n- end_date: The date and time to end selection \
-                            when filtering by time period\
+            \n- start_date: The date and time to start selection in range \
+                            when filtering by the time period\
+            \n- end_date: The date and time to end selection in range \
+                            when filtering by the time period\
             \n- page: Page number to select when paginating\
             \n- per_page: The maximum number of pages per page when paginating")
 
-    @Auth.user_roles('Admin', 'Default User')
+    # all_events = graphene.Field(
+    #     PaginatedDailyRoomEvents,
+    #     start_date=graphene.String(),
+    #     end_date=graphene.String(),
+    #     page=graphene.Int(),
+    #     per_page=graphene.Int(),
+    #     description="Query that returns a list of events given the arguments\
+    #         \n- start_date: The date and time to start selection in range \
+    #                         when filtering by the time period\
+    #         \n- end_date: The date and time to end selection in range \
+    #                         when filtering by the time period\
+    #         \n- page: Page number to select when paginating\
+    #         \n- per_page: The maximum number of pages per page when paginating") # noqa
+
```

Comment:
@koitoror Why do you have som many commented lines?

Reply:
resolved them. thanks.